pkgs/top-level/all-packages.nix

```diff
@@ -7787,6 +7787,8 @@ in
 
   binaryen = callPackage ../development/compilers/binaryen { };
 
+  bluespec-bsc = callPackage ../development/compilers/bluespec-bsc { };
```

Comment:
`s/bluespec-bsc/bluespec/`, imo

Reply:
I wonder, as "bluespec" is the name of the language, not the compiler. I moved it to `bluespec`, but not sure.

Reply:
Yeah, I understand this feeling. Bluespec Inc has always been very forward about allowing people to re-implement the language and even tried to donate it to the SystemVerilog standards committee. So yes you're entirely correct, it's the language name not the name of the compiler project. So, my logic is very trivial and stupid and boils down to: it looks better all around, and I doubt we'll have any competition come along soon considering it's difficult enough to implement. :P

Really the compiler would need some kind of different "working project name" I feel. The only reason `bsc` doesn't work is just because, it's a little too short IMO, though I could be convinced otherwise. Here's a hypothetical: if there wasn't all of `haskellPackages`, and we *only* had a GHC package available, what would the name of that expression be? `ghc` is just as short/cryptic, but does not seem inscrutable at all in contrast. So the argument is a bit flimsy, familiarity with the name is a big part. Maybe they're *both* cryptic, or they're *both* fine names, I'm not sure.

So I think it really just boils down to making the expression discover-able and natural for users who want it, and I think `bluespec` does that well enough without being overly specific or whatever, without anything else making the name difficult to find/remember.

Reply:
`bsc` is already taken in `all-packages.nix` anyways. So works for me.

If a second implementation pops up, let's revisit this ;-)